[2023-06-15T17:52:05.186+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ejecutar_python.check_participants manual__2023-06-15T14:37:45.849041+00:00 [queued]>
[2023-06-15T17:52:05.194+0200] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ejecutar_python.check_participants manual__2023-06-15T14:37:45.849041+00:00 [queued]>
[2023-06-15T17:52:05.194+0200] {taskinstance.py:1308} INFO - Starting attempt 7 of 8
[2023-06-15T17:52:05.207+0200] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): check_participants> on 2023-06-15 14:37:45.849041+00:00
[2023-06-15T17:52:05.212+0200] {standard_task_runner.py:57} INFO - Started process 19499 to run task
[2023-06-15T17:52:05.214+0200] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'ejecutar_python', 'check_participants', 'manual__2023-06-15T14:37:45.849041+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/thesis_container.py', '--cfg-path', '/tmp/tmpahw42heh']
[2023-06-15T17:52:05.215+0200] {standard_task_runner.py:85} INFO - Job 15: Subtask check_participants
[2023-06-15T17:52:05.251+0200] {task_command.py:410} INFO - Running <TaskInstance: ejecutar_python.check_participants manual__2023-06-15T14:37:45.849041+00:00 [running]> on host DESKTOP-3KJU1M7.
[2023-06-15T17:52:05.313+0200] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='ejecutar_python' AIRFLOW_CTX_TASK_ID='check_participants' AIRFLOW_CTX_EXECUTION_DATE='2023-06-15T14:37:45.849041+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-15T14:37:45.849041+00:00'
[2023-06-15T17:52:05.314+0200] {logging_mixin.py:149} INFO - ['hola_525']
[2023-06-15T17:52:05.315+0200] {logging_mixin.py:149} INFO - []
[2023-06-15T17:52:05.315+0200] {logging_mixin.py:149} INFO - ['hola_525']
[2023-06-15T17:52:05.315+0200] {logging_mixin.py:149} INFO - New participants found. Creating containers...
[2023-06-15T17:52:05.315+0200] {logging_mixin.py:149} INFO - 0
[2023-06-15T17:52:05.315+0200] {logging_mixin.py:149} INFO - <class 'int'>
[2023-06-15T17:52:05.315+0200] {logging_mixin.py:149} INFO - ['hola_525']
[2023-06-15T17:52:05.315+0200] {logging_mixin.py:149} INFO - prueba de lista ['hola_525']
[2023-06-15T17:52:05.315+0200] {logging_mixin.py:149} INFO - ['/home/reynel1995/Thesis/host_2/1_6-6.mrxs']
[2023-06-15T17:52:05.315+0200] {logging_mixin.py:149} INFO - container solito hola_525
[2023-06-15T17:52:05.316+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/host_2/1_6-6.mrxs
[2023-06-15T17:52:05.316+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/host_2/1_6-6.mrxs
[2023-06-15T17:52:05.316+0200] {logging_mixin.py:149} INFO - /home/reynel1995/Thesis/host_2
[2023-06-15T17:52:05.316+0200] {logging_mixin.py:149} INFO - nuevo /home/reynel1995/Thesis/hola_525
[2023-06-15T17:52:05.316+0200] {logging_mixin.py:149} INFO - ['/home/reynel1995/Thesis/hola_525']
[2023-06-15T17:52:05.980+0200] {logging_mixin.py:149} INFO - Container hola_525 image built successfully.
[2023-06-15T17:52:06.607+0200] {logging_mixin.py:149} INFO - Container hola_525 started.
[2023-06-15T17:52:06.608+0200] {logging_mixin.py:149} INFO - copy files function ['/home/reynel1995/Thesis/hola_525']
[2023-06-15T17:52:06.608+0200] {logging_mixin.py:149} INFO - hola_525
[2023-06-15T17:52:06.734+0200] {logging_mixin.py:149} INFO - File /home/reynel1995/Thesis/host_1/Image_analysis.py copied to container hola_525 volume.
[2023-06-15T17:53:23.956+0200] {logging_mixin.py:149} INFO - Command 'python /app/Image_analysis.py' executed in container hola_525. Output:
Number of levels in this image are: 10
Dimensions (at native resolution) of various levels in this image are: ((249600, 262912), (124800, 131456), (62400, 65728), (31200, 32864), (15600, 16432), (7800, 8216), (3900, 4108), (1950, 2054), (975, 1027), (487, 513))
Number of levels is downsampled by an amount of: (1.0, 2.0, 4.0, 8.0, 16.0, 32.0, 64.0, 128.0, 256.0, 512.0)
180
[2023-06-15T17:53:23.957+0200] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/reynel1995/thesis/lib/python3.10/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/reynel1995/thesis/lib/python3.10/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/reynel1995/airflow/dags/thesis_container.py", line 303, in check_participants
    execute_command_in_containers("python /app/Image_analysis.py", index)
  File "/home/reynel1995/airflow/dags/thesis_container.py", line 230, in execute_command_in_containers
    for i, filename in enumerate(os.listdir(container_id)):
FileNotFoundError: [Errno 2] No such file or directory: 'hola_525'
[2023-06-15T17:53:24.021+0200] {taskinstance.py:1345} INFO - Marking task as UP_FOR_RETRY. dag_id=ejecutar_python, task_id=check_participants, execution_date=20230615T143745, start_date=20230615T155205, end_date=20230615T155324
[2023-06-15T17:53:24.810+0200] {standard_task_runner.py:104} ERROR - Failed to execute job 15 for task check_participants ([Errno 2] No such file or directory: 'hola_525'; 19499)
[2023-06-15T17:53:24.821+0200] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-15T17:53:24.866+0200] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
